### ISSUE TITLE
installs wrong multiprocess wheel for pypy

### ISSUE THREAD

**Issue by mmckerns** [uqfoundation/pathos#222]
Travis CI installs the wrong multiprocess wheel when for pypy3.6 and pypy3.7.
A `pathos` pypy3.6 install identifies `multiprocess-0.70.12.2-py36-none-any.whl` as a dependency
```
Installed /home/travis/virtualenv/pypy3.6-7.3.3/site-packages/pathos-0.2.9.dev0-py3.6.egg
Processing dependencies for pathos==0.2.9.dev0
Searching for multiprocess>=0.70.12.1
Reading https://pypi.org/simple/multiprocess/
Downloading https://files.pythonhosted.org/packages/df/d0/c3011a5bb77f307e68682a5046cce1a2c6591267bf24b5bf3fc4130bb39d/multiprocess-0.70.12.2-py36-none-any.whl#sha256=b3f866f7d9c7acc1a9cb1b6063a29f5cb140ff545b35b71fd4bfdac6f19d75fa
Best match: multiprocess 0.70.12.2
Processing multiprocess-0.70.12.2-py36-none-any.whl
Installing multiprocess-0.70.12.2-py36-none-any.whl to /home/travis/virtualenv/pypy3.6-7.3.3/site-packages
Adding multiprocess 0.70.12.2 to easy-install.pth file
```
and a `pathos` pypy3.7 install also identifies `multiprocess-0.70.12.2-py36-none-any.whl`
```
Installed /home/travis/virtualenv/pypy3.7-7.3.5/site-packages/pathos-0.2.9.dev0-py3.7.egg
Processing dependencies for pathos==0.2.9.dev0
Searching for multiprocess>=0.70.12.1
Reading https://pypi.org/simple/multiprocess/
Downloading https://files.pythonhosted.org/packages/df/d0/c3011a5bb77f307e68682a5046cce1a2c6591267bf24b5bf3fc4130bb39d/multiprocess-0.70.12.2-py36-none-any.whl#sha256=b3f866f7d9c7acc1a9cb1b6063a29f5cb140ff545b35b71fd4bfdac6f19d75fa
Best match: multiprocess 0.70.12.2
Processing multiprocess-0.70.12.2-py36-none-any.whl
Installing multiprocess-0.70.12.2-py36-none-any.whl to /home/travis/virtualenv/pypy3.7-7.3.5/site-packages
Adding multiprocess 0.70.12.2 to easy-install.pth file
```

Note that both do not install the pypy-specific versions (i.e. `multiprocess-0.70.12.2-pp36-none-any.whl`), and the latter also installs the wrong version of python (i.e. 3.6 instead of 3.7).

**Comment by mmckerns** [comment]
a `pathos` pypy2.7 also installs the wrong `multiprocess` dependency (zip not whl):
```
Installed /home/travis/virtualenv/pypy2.7-7.3.1/site-packages/pathos-0.2.9.dev0-py2.7.egg
Processing dependencies for pathos==0.2.9.dev0
Searching for multiprocess>=0.70.12.1
Reading https://pypi.org/simple/multiprocess/
Downloading https://files.pythonhosted.org/packages/99/1a/472900644359cdd208d1fbe71706bdeecbc6e8db2e39c35ebe89459e9172/multiprocess-0.70.12.2.zip#sha256=206bb9b97b73f87fec1ed15a19f8762950256aa84225450abc7150d02855a083
Best match: multiprocess 0.70.12.2
Processing multiprocess-0.70.12.2.zip
Writing /tmp/easy_install-ZZ7SxN/multiprocess-0.70.12.2/setup.cfg
Running multiprocess-0.70.12.2/setup.py -q bdist_egg --dist-dir /tmp/easy_install-ZZ7SxN/multiprocess-0.70.12.2/egg-dist-tmp-aEPlAM
no previously-included directories found matching '.git'
no previously-included directories found matching '.coverage'
creating /home/travis/virtualenv/pypy2.7-7.3.1/site-packages/multiprocess-0.70.12.2-py2.7.egg
Extracting multiprocess-0.70.12.2-py2.7.egg to /home/travis/virtualenv/pypy2.7-7.3.1/site-packages
Adding multiprocess 0.70.12.2 to easy-install.pth file

```

**Comment by mmckerns** [comment]
Same result for local install with pypy3.6
```
Installed /Users/mmckerns/lib/pypy-3.6/site-packages/pathos-0.2.9.dev0-py3.6.egg
Processing dependencies for pathos==0.2.9.dev0
Searching for multiprocess>=0.70.12.1
Reading https://pypi.org/simple/multiprocess/
Downloading https://files.pythonhosted.org/packages/df/d0/c3011a5bb77f307e68682a5046cce1a2c6591267bf24b5bf3fc4130bb39d/multiprocess-0.70.12.2-py36-none-any.whl#sha256=b3f866f7d9c7acc1a9cb1b6063a29f5cb140ff545b35b71fd4bfdac6f19d75fa
Best match: multiprocess 0.70.12.2
Processing multiprocess-0.70.12.2-py36-none-any.whl
...
```
and pypy3.7
```
Installed /Users/mmckerns/lib/pypy-3.7/site-packages/pathos-0.2.9.dev0-py3.7.egg
Processing dependencies for pathos==0.2.9.dev0
Searching for multiprocess>=0.70.12.1
Reading https://pypi.org/simple/multiprocess/
Downloading https://files.pythonhosted.org/packages/df/d0/c3011a5bb77f307e68682a5046cce1a2c6591267bf24b5bf3fc4130bb39d/multiprocess-0.70.12.2-py36-none-any.whl#sha256=b3f866f7d9c7acc1a9cb1b6063a29f5cb140ff545b35b71fd4bfdac6f19d75fa
Best match: multiprocess 0.70.12.2
Processing multiprocess-0.70.12.2-py36-none-any.whl
...
```

**Comment by mmckerns** [comment]
should be fixed by the release of `multiprocess-0.70.13`
